README.md

```diff
@@ -1 +1,63 @@
-# Hello-Microverse
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a simple project without any coding complex built with html and css that display a hello microverse message in your browser.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
```

Comment:
- [ ] I kindly suggest that you remove `React.js`, `Express.js` and `Database` from your tech stack and replace them with the stack you made use of in this project e.g HTML, CSS.